src/main/docs/guide/micronautOpenApiAnnotations/openApiRequest.adoc

```diff
@@ -0,0 +1,73 @@
+Annotation api:openapi.annotation.OpenAPIRequest[] can be used as alternative to *io.micronaut.http.annotation.RequestBean* to mark parameter as wrapped request details. It useful, when you haven't micronaut-http in dependencies. For example, when you use micronaut-openapi with spring-boot application.
+
+.Example of using `@OpenAPIRequest` annotation with a Spring Boot application
+[source,java]
+----
+@RestController
+@RequestMapping("/my-api")
+public class MyController {
```

Comment:
this should be an include pulled from sources with a kotlin example

Reply:
How can I make it so that there are examples from Kotlin and Java at the same time?